samples/src/main/python/SynapseSample.py

```diff
@@ -0,0 +1,122 @@
+# Optional:
```

Comment:
Should we explain that this is not the repository that holds the synapse integration code? we could say that the other package utilizes this one and it is private